lib/oxsdatatypes/README.md

```diff
@@ -32,6 +32,13 @@ Each datatype provides:
 * `from_be_bytes` and `to_be_bytes` methods for serialization.
 
 
+## Calendar value now operation
+
+The `DateTime::now()` function needs special OS support.
+Currently:
+- For `wasm32-unknown-unknown` if the `js` feature is enabled the `Date.now()` ECMAScript API is used.
+- For all other targets `SystemTime::now()` is used.
```

Comment:
Shouldn't the option to use a custom now function via the `custom-now` target be the third bullet point here?

Reply:
Indeed. I forgot about it.